pkg/auth/auth.go

```diff
@@ -18,8 +18,15 @@ package auth
 
 import (
 	"context"
+
+	"github.com/libopenstorage/openstorage/pkg/correlation"
 )
 
+func init() {
+	correlation.RegisterComponent("auth")
```

Comment:
do you think this should be `osd/auth` as a pkg to make it more unique and void conflicts with other `auth` packages?

Reply:
Yep makes sense